Document Snaps bundle analyzer option (#1955)
MetaMask/metamask-docs: snaps/reference/cli/subcommands.md

`b` is an alias for `build`.

#### `analyze`

<Tabs>
<TabItem value="Syntax">

```bash
yarn mm-snap build --analyze
```

</TabItem>
</Tabs>

> Enables analyzing the Snap bundle.

Reviewer: Should we add information here? What gets produced when we use this? Why would we use this? 
Do we need to reference this anywhere in the Snaps content?

`b` is an alias for `build`.

#### `analyze`

<Tabs>
<TabItem value="Syntax">

```bash
yarn mm-snap build --analyze
```

</TabItem>
</Tabs>

Enables analyzing the Snap [bundle](../../learn/about-snaps/files.md#bundle-file).
This uses [`webpack-bundle-analyzer`](https://github.com/webpack-contrib/webpack-bundle-analyzer) under the hood,
which creates an interactive visualization of the contents of your bundle.

> The visualization is located at the port displayed in the command line output.

Reviewer: Does this mean something like `http://localhost:<port>`>

Author: Yes - maybe "URL" is more clear?

`b` is an alias for `build`.

#### `analyze`

<Tabs>
<TabItem value="Syntax">

```bash

> yarn mm-snap build --analyze <BOOLEAN>

Reviewer 2: It technically takes a boolean value, but since it's disabled by default, it's easier to think of the flag as enabling the bundle analyser. So just specifying this will enable it:

```suggestion
yarn mm-snap build --analyze
```

```

</TabItem>
<TabItem value="Example">

```bash
yarn mm-snap build --analyze false
```

</TabItem>
</Tabs>

> Enables or disables analyzing the Snap bundle.

Reviewer 2: ```suggestion
Enables analyzing the Snap bundle.
```

This uses [`webpack-bundle-analyzer`](https://github.com/webpack-contrib/webpack-bundle-analyzer) under the hood.

:::info

> This option requires [`@metamask/create-snap`](https://github.com/MetaMask/snaps/tree/main/packages/create-snap) version 4.0.4 or later.

Reviewer 2: ```suggestion
This option requires [`@metamask/snaps-cli`](https://github.com/MetaMask/snaps/tree/main/packages/snaps-cli) version 6.7.0 or later.
```
